app/views/teaching_events/about_git_events.html.erb

```diff
@@ -28,8 +28,16 @@
       <li>find out how to apply for teacher training</li>
       <li>explore what funding is available</li>
     </ul>
-  </section>
-
+<p>Watch our video to see what happens at a Get Into Teaching event and how it could help you make a decision.</p>
```

Comment:
```suggestion
<p>Watch our video to see what happens at a Get Into Teaching event and how it could help you make a decision about teaching.</p>
```
Worth giving more context about the decision?